ui/components/app/transaction-breakdown/transaction-breakdown.component.js

```diff
@@ -139,22 +148,28 @@ export default class TransactionBreakdown extends PureComponent {
             )}
           </TransactionBreakdownRow>
         )}
-        <TransactionBreakdownRow
-          title={t('transactionHistoryEffectiveGasPrice')}
-        >
-          <UserPreferencedCurrencyDisplay
-            className="transaction-breakdown__value transaction-breakdown__value--effective-gas-price"
-            type={PRIMARY}
-            value={effectiveGasPrice}
-          />
-          {showFiat && (
+        {process.env.SHOW_EIP_1559_UI && supportsEIP1559 && (
```

Comment:
We can remove this `process.env.SHOW_EIP_1559_UI` check here, but that can be done in another PR.